src/deluge/io/midi/midi_device.cpp

```diff
@@ -117,8 +117,8 @@ resetBendRanges
 				soundEditor.mpeZonesPotentiallyUpdated();
 			}
 
-			// Upper zone
-			else if (channel == 15) {
+			// Master Channel of Upper zone
+			else if (ports[MIDI_DIRECTION_INPUT_TO_DELUGE].mpeUpperZoneLastMemberChannel < 15 && channel == 15) {
```

Comment:
If mpeUpperZoneLastMemberChannel is lower than 15 it means that "MPE Upper Zone" is enabled and then it is true that channel 15 is a master channel

Reply:
Can this also use is master channel?

Reply:
No because there is an ifelse that distinguishes between channel 0 and channel 15 which do different things and isMasterChannels checks for any of them